web/viewer.css

```diff
@@ -1471,6 +1471,14 @@ dialog :link {
   }
 }
 
+:is(.xfaRight) > :is(.xfaCaption, .xfaCaptionForCheckButton) {
+  margin-left: 4px;
+}
+
+:is(.xfaLeft) > :is(.xfaCaption, .xfaCaptionForCheckButton) {
+  margin-right: 4px;
+}
+
```

Comment:
This is unfortunately not the correct file for XFA-specific CSS rules, please see https://github.com/mozilla/pdf.js/blob/master/web/xfa_layer_builder.css

Reply:
https://github.com/mozilla/pdf.js/blob/master/web/xfa_layer_builder.css#L156-L158

Reply:
I've moved the style to the correct stylesheet. Thank you!